node-util/sbin/oo-idler-stats

```diff
@@ -131,7 +130,7 @@ def percent(a, b):
 
 
 def cartridge_types(homedir):
-    return_code, types = run("ls |egrep -v '-|git'", homedir)
+    return_code, types = run("ls |egrep -v -e '-|git'", homedir)
```

Comment:
Could you explain why this change was necessary? The previous syntax looks correct according to the egrep manpage and some quick testing.